.gitbook/contents/developer/boba-basics/bridge-basics/boba-token-bridge.md

```diff
@@ -8,15 +8,11 @@ The BOBA token bridge functionality provides a method for the BOBA token to be d
 
 The BOBA Token Bridge is composed of two main contracts the [`EthBridge` (opens new window)](https://github.com/bobanetwork/boba\_legacy/blob/develop/packages/boba/contracts/contracts/lzTokenBridge/EthBridge.sol)(for Ethereum) and the [`AltL1Bridge` (opens new window)](https://github.com/bobanetwork/boba/blob/develop/packages/boba\_legacy/contracts/contracts/lzTokenBridge/AltL1Bridge.sol)(for Alt L1s).
 
-> Note: **To protect our users, we are only allowed to bridge BOBA tokens between Ethereum and Alt L1s.** For example, you are not allowed to directly bridge BOBA tokens from Moonbeam to BNB. What you can do is to bridge it from Moonbeam to Etherem and bridge it to BNB.
```

Comment:
Maybe as a replacement we can add something in the form of - "To protect users, theses bridges only allow to bridge .... However, the light bridge offers an alt option to ..."

Reply:
I can add this back, though I don't really understand what it's documenting, since fundamentally, with only Boba Eth and Boba BNB, there is no combination of L1s where the bridging does not work.  So I don't understand how we can say "To protect our users, we don't allow you to do this thing that doesn't actually exist".

I was trying to come up with a sentence to incorporate the light bridge, but there is no Eth Mainnet <-> BOBA BNB, nor Eth Mainnet <-> BNB Mainnet,  as I read in the docs at least.  So, I'm having trouble making it make sense in this context.

Any ideas?

Reply:
true - this needs some fixing. As i see it the line mostly wants to inform users that these bridges (EthBridge/AltL1Bridge) are setup (and meant to be) between Ethereum and BNB (L1) (because its the canonical way). Ethereum <> L2 should not be supported because of the finality on L2, and BNB <> Boba ETH wouldn't work either (because tokens have a source at Ethereum)

Reply:
Given that this section is titled:

> # Using the BOBA Token Bridge Between L1s

I feel like it's clear that this path is not intended for anything related to L2 bridging.

I do think this section could be improved, but my feeling is that removing is the clearest option for the moment.

Reply:
sure thing! makes sense